src/toolbox.js

```diff
@@ -2797,6 +2797,11 @@ export default (favorites) => {
             <block type="generate_image_openai"/>
             <block type="text_completion_openai" />
         </category>
+        <category name="Fsh API" colour="#50494e">
+            <label text="Fsh api"/>
+            <label text="Image api"/>
+            <block type="inv_fsh_api_animal"/>
+        </category>
```

Comment:
Is the toolbox category which in those blocks are merged okay?

Reply:
ye?

Reply:
yes

Reply:
> ye?

That's for lead devs to answer.

Your edit works, that's not of debate;
the debate is "is it where it should be?"

Reply:
at the bottom of what because its at the bottom of applications already

Reply:
> at the bottom of what because its at the bottom of applications already

Ye, didn't noticed.